src/main/java/io/github/thebusybiscuit/slimefun4/core/services/PerWorldSettingsService.java

```diff
@@ -196,20 +196,27 @@ public void save(@Nonnull World world) {
             return optional.get();
         } else {
             Set<String> items = new LinkedHashSet<>();
-            Config config = getConfig(world);
 
-            config.getConfiguration().options().header("This file is used to disable certain items in a particular world.\nYou can set any item to 'false' to disable it in the world '" + name + "'.\nYou can also disable an entire addon from Slimefun by setting the respective\nvalue of 'enabled' for that Addon.\n\nItems which are disabled in this world will not show up in the Slimefun Guide.\nYou won't be able to use these items either. Using them will result in a warning message.");
-            config.getConfiguration().options().copyHeader(true);
-            config.setDefaultValue("enabled", true);
+            if (!SlimefunPlugin.getCfg().getBoolean("options.disable-per-world-config")) {
```

Comment:
There is no need to read this boolean everytime a World is loaded.
The value should probably be stored as a field and read via the `load()` method.

Reply:
ok